tier4_map_msgs/CMakeLists.txt

```diff
@@ -0,0 +1,28 @@
+cmake_minimum_required(VERSION 3.5)
+project(tier4_map_msgs)
+
+if(NOT CMAKE_CXX_STANDARD)
+  set(CMAKE_CXX_STANDARD 14)
+  set(CMAKE_CXX_STANDARD_REQUIRED ON)
+  set(CMAKE_CXX_EXTENSIONS OFF)
+endif()
+
+if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+  add_compile_options(-Wall -Wextra -Wpedantic)
+endif()
```

Comment:
Message package does not need c++ options.

Reply:
removed at [6fb7829](https://github.com/tier4/tier4_autoware_msgs/pull/80/commits/6fb78294cac8d9aff853231ae0e20a6b0f1aacdd)